docs/user-guide/nextcloud-integration/README.md

```diff
@@ -0,0 +1,85 @@
+---
+sidebar_navigation:
+  title: Nextcloud integration
+  priority: 600
+description: Nextcloud integration
+keywords: integrations, apps, Nextcloud
+
+---
+
+# Nextcloud integration
+
+[Nextcloud](https://nextcloud.com/), the world’s most-deployed on-premises collaborative file storage platform and [OpenProject](https://www.openproject.org/), the leading free and open source project management and collaboration software, join forces. **Data sovereignty** and **open source** are core values that are important to both OpenProject and Nextcloud and which firn the foundational common ground for this integration.
```

Comment:
I reduced this bit because I was afraid it would otherwise sound too press release-y.